src/main/java/org/folio/linked/data/model/entity/Resource.java

```diff
@@ -176,4 +173,30 @@ public Resource addIncomingEdge(@NonNull ResourceEdge incomingEdge) {
     return this;
   }
 
+  public UUID getInstanceInventoryId() {
```

Comment:
TODO (I will do this on Monday as part of this PR) - Remove the getter methods getInstanceInventoryId  and getInstanceSrsId  after replacing "instanceId" and "srsId" in the response DTO with "instanceMetadata" DTO.

Reply:
Done